components/dependency_injection/definitions.rst

```diff
@@ -32,34 +35,68 @@ with these methods and make changes to it these will be reflected in the
 container. If, however, you are creating a new definition then you can add
 it to the container using::
 
-    $container->setDefinition($id, $definition);
+    use Symfony\Component\DependencyInjection\Definition;
+
+    $definition = new Definition('Acme\Service\MyService');
+    $container->setDefinition('app.my_service', $definition);
```

Comment:
Let's keep `acme` for the components section for now (even more recently added parts of this section consistently use Acme).